OpenRA.Mods.Common/Traits/Health.cs

```diff
@@ -23,6 +23,7 @@ public class HealthInfo : ITraitInfo, UsesInit<HealthInit>
 		public readonly bool NotifyAppliedDamage = true;
 
 		[FieldLoader.LoadUsing("LoadShape")]
+		[Desc("Capsule, Circle and Rectangle are supported.")]
```

Comment:
The properties of the three shapes aren't documented (#7913), so I don't see any benefit from this.